dbms/src/Storages/DeltaMerge/Delta/DeltaValueSpace.h

```diff
@@ -434,6 +436,40 @@ class DeltaValueInputStream : public IBlockInputStream
     String getName() const override { return "DeltaValue"; }
     Block getHeader() const override { return persisted_files_input_stream.getHeader(); }
 
+    bool getSkippedRows(size_t &) override { throw Exception("Not implemented", ErrorCodes::NOT_IMPLEMENTED); }
+
+    /// Skip next block in the stream.
+    /// Return false if failed to skip or the end of stream.
+    bool skipNextBlock(size_t skip_rows) override
```

Comment:
I think it's better to be like `size_t skipNextBlock()`;

As the function name said, you want to skip a block, we don't need to know how many rows we need skip.
Instead, we can return with the skip rows, for the further assert check.
Besides, for most functions about skipNextBlock, we don't use the parameter skip_rows, so it's a little wired.

Reply:
make sense, and done.